locale/cups_de.po

```diff
@@ -4443,22 +4443,22 @@ msgid "US Legal"
 msgstr "US Lang"
 
 msgid "US Legal Oversize"
-msgstr ""
+msgstr "US Legal übergross"
 
 msgid "US Letter"
 msgstr "US Letter"
 
 msgid "US Letter Long Edge"
-msgstr ""
+msgstr "US Letter lange Seite"
 
 msgid "US Letter Oversize"
-msgstr ""
+msgstr "US Letter übergross"
 
 msgid "US Letter Oversize Long Edge"
-msgstr ""
+msgstr "US Letter übergross, lange Seite"
```

Comment:
should be "übergroß" not "übergross" if this is not considered to be Swiss german